travis/licenses_golden/licenses_third_party

```diff
@@ -4624,12 +4624,19 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 LIBRARY: dart
 ORIGIN: ../../../third_party/dart/LICENSE
 TYPE: LicenseType.bsd
+FILE: ../../../third_party/dart/.gdb_history
```

Comment:
I think you need to purge some files from your `third_party/dart` namely `.gdb_history`, `*.swp`, `*.orig`, `*.rej`.

Reply:
Done